lib/services/project-changes-service.ts

```diff
@@ -112,9 +120,11 @@ export class ProjectChangesService implements IProjectChangesService {
 			this._prepareInfo.bundle = projectChangesOptions.bundle;
 		}
 		if (this._changesInfo.packageChanged) {
+			this.$logger.trace("Set modulesChanged to true as packageChanged is true");
 			this._changesInfo.modulesChanged = true;
 		}
 		if (this._changesInfo.modulesChanged || this._changesInfo.appResourcesChanged) {
+			this.$logger.trace(`Set configChanged to true, current value of moduleChanged is: ${this._changesInfo.modulesChanged}, appResourcesChanged is: ${this._changesInfo.appResourcesChanged}`);
```

Comment:
Maybe it is better to print all changesInfo.

Reply:
It is printed at the end of the method. Or you want to know the value of the properties before setting all of them to true?

Reply:
Actually I just want to see all changesInfo and didn't notice it is printed at the end of the method.